docs/content/3.manual/manual.yml

```diff
@@ -408,15 +412,16 @@ sections:
           Like `.[]`, but no errors will be output if . is not an array
           or object.
 
-      - title: "`,`"
+      - title: "Comma, Output Stream Join: `,`"
         body: |
 
           If two filters are separated by a comma, then the
-          input will be fed into both and there will be multiple
-          outputs: first, all of the outputs produced by the left
-          expression, and then all of the outputs produced by the
-          right. For instance, filter `.foo, .bar`, produces
-          both the "foo" fields and "bar" fields as separate outputs.
+          same input will be fed into both and the two filters' output
+          value streams will be joined: first, all of the outputs
+          produced by the left expression, and then all of the outputs
+          produced by the right. For instance, filter `.foo, .bar`,
+          produces both the "foo" fields and "bar" fields as separate
+          outputs.
 
         examples:
```

Comment:
Should include an example that makes it clear that the next step in the pipeline is called separately (and sequentially) with each value. Ex: `'["a","b","c","d"]' >>> '.[1],.[3] | [.]' >>> ["b"],["d"]`